app/src/ui/diff/index.tsx

```diff
@@ -64,6 +64,11 @@ interface IDiffProps {
 
   /** Hiding whitespace in diff. */
   readonly hideWhitespaceInDiff: boolean
+
+  readonly onDiscardChanges?: (
```

Comment:
Can we copy and paste the nice documentation you added for `onDiscardChanges` in `ITextDiffProps` here?